Core/Services/StitchCoreRemoteMongoDBService/Tests/StitchCoreRemoteMongoDBServiceTests/Sync/DataSynchronizerUnitTests.swift

```diff
@@ -114,4 +117,84 @@ class DataSynchronizerUnitTests: XCMongoMobileTestCase {
 
         XCTAssertFalse(dataSynchronizer.isRunning)
     }
+
+    func testCount() throws {
+        dataSynchronizer.configure(namespace: namespace,
+                                   conflictHandler: TestConflictHandler(),
+                                   changeEventListener: TestEventListener(),
+                                   errorListener: TestErrorListener())
+        XCTAssertEqual(0, try dataSynchronizer.count(in: namespace))
+
+        let doc1 = ["hello": "world", "a": "b"] as Document
+        let doc2 = ["hello": "computer", "a": "b"] as Document
+
+        try XCMongoMobileTestCase.client.db(DataSynchronizer.localUserDBName(withInstanceKey: instanceKey.oid, for: namespace))
```

Comment:
[nit] If we're going to be getting a pure local collection like this for all of our CRUD tests, it might be worth adding a helper method to XCMongoMobileTestCase that will get the collection, and we should then reuse the result throughout each test rather than getting it every time we want to do a CRUD operation